### PR TITLE
Delete my library

### PR DESCRIPTION
**Pull request by lhrDev**
Hello, I would like to remove my library to address several errors that I overlooked upon its initial publication. My intention is to re-publish it once these issues have been resolved. Thank you for your understanding.

### CODE REVIEW

**Comment by lhrDev** [comment]
???

**Review by per1234 — COMMENTED** [review]
Hello @lhrDev. I am confirming your request.

Unlike submissions, this type of request is still handled manually. The Library Manager index maintainer occasionally comes through to clear all the outstanding requests. They will merge this pull request when it is done. You will get a notification from GitHub at that time.

**Review by MatteoPologruto — APPROVED** [review]
Done:
```
2024/01/11 14:11:10 Loaded 6531 libraries from DB
Removing OtaHelper
Success!
```

**Pull request merged**
